database/getters.go

```diff
@@ -20,6 +22,12 @@ func (g *Get) AllTableNames() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer func(rows *sql.Rows) {
+		err := rows.Close()
+		if err != nil {
+			log.Println("Error closing rows:", err)
```

Comment:
Should we log this? Or should we return as err? We cant `return nil, err` here as we already got the table names value. What do you think will be a good approach to this.

Reply:
I added logging for now. Im not quite sure what is happening behind the scenes when we get an error closing it and keep going on.

Reply:
I guess it doesnt matter. [docs[(https://pkg.go.dev/database/sql#Rows.Close)